Packages/Tracking/Physical Hands/Runtime/Scripts/UI/PhysicalHandsButtonBase.cs

```diff
@@ -0,0 +1,479 @@
+/******************************************************************************
+ * Copyright (C) Ultraleap, Inc. 2011-2024.                                   *
+ *                                                                            *
+ * Use subject to the terms of the Apache License 2.0 available at            *
+ * http://www.apache.org/licenses/LICENSE-2.0, or another agreement           *
+ * between Ultraleap and you, your company or other organization.             *
+ ******************************************************************************/
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Leap.Unity.PhysicalHands
+{
+    [RequireComponent(typeof(Rigidbody))]
+    public class PhysicalHandsButtonBase : MonoBehaviour
+    {
+
```

Comment:
We should check that all non-private variables/functions NEED that access level

Reply:
I think i got everything